packages/wallet-adapter-core/src/LegacyWalletPlugins/conversion.ts

```diff
@@ -44,6 +44,25 @@ export function convertV2PayloadToV1JSONPayload(
   if ("bytecode" in payload) {
     // is a script payload
     throw new Error("script payload not supported");
+    // is multisig funtion payload
+  } else if ("multisigAddress" in payload) {
+    const stringTypeTags: string[] | undefined = payload.typeArguments?.map(
+      (typeTag) => {
+        if (typeTag instanceof TypeTag) {
+          return typeTag.toString();
+        }
+        return typeTag;
+      }
```

Comment:
FYI, then probably can just do `toString()` and let the string convert to itself (I think)